lib/xlsx/xlsx.js

```diff
@@ -285,9 +280,11 @@ class XLSX {
           entryName = entryName.substr(1);
         }
         let stream;
-        if (entryName.match(/xl\/media\//) ||
+        if (
+          entryName.match(/xl\/media\//) ||
```

Comment:
which issue is it fix?

Reply:
I believe this was just from running the project linter, if not I can revert this